### ISSUE TITLE
Add customer filter to the project evaluation report

### ISSUE THREAD

**Issue by jaragunde** [Igalia/phpreport#290]
There is currently no way to search projects by their customer name unless that name is part of the project description. We should add a customer field to the project evaluation report.

**Comment by tonythomas01** [comment]
This is fixed in #299

**Comment by jaragunde** [comment]
```
commit 787d548038f6b6d102f865fbe79e186677614d70
Merge: 8e2fdd5 be60dd7
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Tue Sep 27 20:23:16 2016 +0200

    [#186][#290] Add customer field in several reports
    
    (Merge pull request #299 from tonythomas01/add-customer-listing)

commit be60dd72f00581e6dd1c2007b6127a9e8b37718f
Author: Tony Thomas <01tonythomas@gmail.com>
Date:   Sat Sep 10 14:00:03 2016 +0530

    [#290] Add customer filter to the project evaluation report
